app/views/idv/in_person/usps_list.html.erb

```diff
@@ -33,7 +33,7 @@
           FORT MYER, VA 22211-9998 </div>
       </div>
     </label>
-    <label class="btn-border col-12 mb2 hide" for="two_factor_options_form_selection_webauthn">
+    <label class="btn-border col-12 mb2 display-none" for="two_factor_options_form_selection_webauthn">
```

Comment:
It's not clear to me why we hide this, but given that the content of the page appears to be static text content for a work-in-progress flow presumably copied from 2FA options page markup, I'm not too concerned about it.